exp-models/addon/models/experiment.js

```diff
@@ -149,6 +153,12 @@ export default DS.Model.extend(JamModel, {
                 [`jam-${config.JAMDB.namespace}:accounts-*`]: 'CREATE'
             }
         });
-        collection.save();
+        this.set('_sessionCollection', collection);
+        collection.save().then(() => {
+            this.set('sessionCollectionId', this.get('sessionCollectionId'));
```

Comment:
`sessionCollectionId` is a computed, and does not depend on the newly created session. Why are you trying to set it?